### ISSUE TITLE
Acitivty list for pre-trained models

### ISSUE THREAD

**Issue by ashwinvaswani** [facebookresearch/SlowFast#251]
Could you please specify on which version of kinetics (400/600/700) were the models in the zoo trained on? 
Also, can you release the activity list (action name to class label conversion) for the set?
 I was evaluating kinetics 700 on the validation set using the Kinetics/c2/SLOW_8x8_R50 config file. However, since I do not know the activity list for kinetics 700 or even 600 for that matter, evaluating scores are poor (almost random). Can you please release the activity list?
